swap_meet/clothing.py

```diff
@@ -1,2 +1,17 @@
-class Clothing:
-    pass
\ No newline at end of file
+from uuid import uuid4 
+from .item import Item
+
+class Clothing(Item): # Updated class to inherit from Item
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        self.id = uuid4().int if id is None else id
+        self.fabric = fabric if fabric != "Unknown" else "Unknown"
+        super().__init__(id, condition)  # pass condition to Item
```

Comment:
Prefer the parent constructor execute first and then set the child class's custom `fabric` attribute

```suggestion
        super().__init__(id, condition)  # pass condition to Item
        self.fabric = fabric if fabric != "Unknown" else "Unknown"
```

Reply:
Noted, thanks! Moved the parent constructor above the child's custom attributes for all 3 child modules.